src/synapse/azext_synapse/_client_factory.py

```diff
@@ -4,6 +4,25 @@
 # --------------------------------------------------------------------------------------------

```

Comment:
Do we need this?

Reply:
Yes

---
In reply to: [416229371](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r416229371) [](ancestors = 416229371)